packages/lexical/src/nodes/LexicalTextNode.ts

```diff
@@ -1196,7 +1196,7 @@ function convertTextDOMNode(domNode: Node): DOMConversionOutput {
     ) {
       const previousTextContent = previousText.textContent || '';
       if (previousTextContent.length > 0) {
-        if (previousTextContent.match(/(?:\s|\r?\n|\t)$/)) {
+        if (previousTextContent.match(/( |\t|\r?\n)$/)) {
```

Comment:
I deliberately dropped the non-capturing specifier here since we don't appear to need it.

Reply:
It doesn't matter if `\n` is preceded with `\r`. So it can be rewritten this way:
```suggestion
        if (previousTextContent.test(/[ \t\n]$/)) {
```